src/tests/test-profile/thinkpad-p1-no-iommu.json

```diff
@@ -0,0 +1,272 @@
+{
+  "SecurityAttributes" : [
+    {
+      "AppstreamId" : "org.fwupd.hsi.Kernel.Tainted",
+      "HsiLevel" : 0,
+      "HsiResult" : "not-tainted",
+      "Plugin" : "linux_tainted",
+      "Flags" : [
+        "success",
+        "runtime-issue"
+      ]
+    },
+    {
+      "AppstreamId" : "org.fwupd.hsi.Kernel.Lockdown",
+      "HsiLevel" : 0,
+      "HsiResult" : "enabled",
+      "Plugin" : "linux_lockdown",
+      "Flags" : [
+        "success",
+        "runtime-issue"
+      ]
+    },
+    {
+      "AppstreamId" : "org.fwupd.hsi.Kernel.Swap",
+      "HsiLevel" : 0,
+      "HsiResult" : "encrypted",
+      "Plugin" : "linux_swap",
+      "Flags" : [
+        "success",
+        "runtime-issue"
+      ]
+    },
+    {
+      "AppstreamId" : "org.fwupd.hsi.Uefi.SecureBoot",
+      "HsiLevel" : 0,
```

Comment:
I think you should strip HsiLevel in this data, let fwupd calculate it by mapping HsiLevel over to AppstreamId internally.  This also lets us re-define which levels things should be tied to based on fwupd version.

Reply:
Okay, lets do this bit first; PR incoming.

Reply:
https://github.com/fwupd/fwupd/pull/4835